kortex_driver/src/non-generated/driver/kortex_arm_driver.cpp

```diff
@@ -526,6 +527,69 @@ void KortexArmDriver::startActionServers()
     }
 }
 
+void KortexArmDriver::moveArmWithinJointLimits()
+{
+    auto joint_angles = m_base->GetMeasuredJointAngles();
+
+    std::map<int, float> limited_joints;
+    if (m_degrees_of_freedom == 6)
+    {
+        // We add angle limitations for joints 1,2 and 4 on 6 dof
+        limited_joints[1] = 128.9;
```

Comment:
is there a variable holding these values? instead of magic numbers?

Reply:
I don't think so. I got them from the User Guide, p.86.